pkg/cluster/internal/create/create.go

```diff
@@ -122,6 +124,12 @@ func Cluster(logger log.Logger, p providers.Provider, opts *ClusterOptions) erro
 			kubeadmjoin.NewAction(),                   // run kubeadm join
 			waitforready.NewAction(opts.WaitForReady), // wait for cluster readiness
 		)
+		// prepare stargz snapshotter serviceaccount if needed
+		if v, ok := os.LookupEnv("KIND_EXPERIMENTAL_STARGZ"); ok && v == "1" {
```

Comment:
are there plans to enable this snapshotter by default eventually?
if so maybe add a TODO + linking to an issue on top of this line.

Reply:
should this PR also touch the kind documentation?